tests/envs/test_blocks.py

```diff
@@ -111,3 +115,66 @@ def test_blocks_failure_cases():
     act = Stack.ground([robot, block0], np.zeros(0)).policy(state)
     next_state = env.simulate(state, act)
     assert state.allclose(next_state)
+
+
+def test_blocks_load_task_from_json():
+    """Tests for loading blocks test tasks from a JSON file."""
+    # Set up the JSON file.
+    task_spec = {
+        "problem_name":
+        "blocks_test_problem1",
+        "blocks": [{
+            "block_id": 0,
```

Comment:
I was wondering but maybe we should give blocks more descriptive IDs so its easy for us to read and debug what's going on?

e.g. `block_id: "red_block"`